--- a/src/plus/gk/utils/checkin.utils.ts	2026-05-28 03:46:35.846496489 +0000
+++ b/src/plus/gk/utils/checkin.utils.ts	2026-05-28 03:46:58.032036523 +0000
@@ -145,6 +145,21 @@
 		effective = { ...actual };
 	}
 
+	const enterprisePlan = getSubscriptionPlan(
+		'enterprise',
+		false,
+		0,
+		undefined,
+		data.user.firstGitLensCheckIn != null
+			? new Date(data.user.firstGitLensCheckIn)
+			: data.user.createdDate != null
+			? new Date(data.user.createdDate)
+			: undefined,
+		undefined,
+		undefined,
+		data.nextOptInDate,
+	);
+
 	let activeOrganization: Organization | undefined;
 	if (organizationId != null) {
 		activeOrganization = organizations.find(organization => organization.id === organizationId);
@@ -156,8 +171,8 @@
 
 	return {
 		plan: {
-			actual: actual,
-			effective: effective,
+			actual: enterprisePlan,
+			effective: enterprisePlan,
 		},
 		account: account,
 		activeOrganization: activeOrganization,
